app/src/main/java/org/p2p/wallet/newsend/repository/RecipientsDatabaseRepository.kt

```diff
@@ -4,17 +4,19 @@ import org.p2p.wallet.newsend.db.RecipientsDao
 import org.p2p.wallet.newsend.model.RecipientConverter
 import org.p2p.wallet.newsend.model.SearchResult
 import java.util.Date
+import org.p2p.wallet.utils.UsernameFormatter
 
 class RecipientsDatabaseRepository(
-    private val recipientsDao: RecipientsDao
+    private val recipientsDao: RecipientsDao,
+    private val usernameFormatter: UsernameFormatter
 ) : RecipientsLocalRepository {
```

Comment:
I'm not sure about formatter inside repository, but moving it elsewhere would require more refactoring